municipal_finance/static/javascript/table.js

```diff
@@ -714,22 +722,31 @@
       var table = this.$('.values').empty()[0];
       var functions = this.functionHeadings();
 
+      var muniColumns = 1;
+      if (cube.model.can_aggregate) {
+        muniColumns = Object.keys(cube.model.aggregate_columns).length;
+        var aggregateColumns = cube.model.aggregate_columns;
+      }
+      else if (cube.columns.length > 1) {
+        muniColumns = cube.columns.length;
+        var aggregateColumns = cube.model.measures;
+      }
```

Comment:
It's nicer to declare the variable once and then assign to it in the if context. It's a bit weird declaring a variable in two places, also to be declaring it in a deeper context than where you use it later.

```suggestion
      var muniColumns = 1;
      var aggregateColumns;
      if (cube.model.can_aggregate) {
        muniColumns = Object.keys(cube.model.aggregate_columns).length;
        aggregateColumns = cube.model.aggregate_columns;
      }
      else if (cube.columns.length > 1) {
        muniColumns = cube.columns.length;
        aggregateColumns = cube.model.measures;
      }
```

Reply:
Included this change in the latest commit